### ISSUE TITLE
Saving Blogs and Discussions - As a User

### ISSUE THREAD

**Issue by YunusDange** [arshadansari27/fitranginew#228]
There is a little confusion when user write any Blog or discussion as ......due to our modification in real-time refresh and all...no message come which should indicate user that the blog or discussion is submitted....it has to be corrected 

the correct way is...once the blog or discussion get submitted properly...the user should automatically leave the editing window (additionally if he receive a pop-up message indicating that his blog/discussion is get submitted and waiting admin approval then that will look perfect)

It's critical

**Comment by YunusDange** [comment]
Inspite of knowing the functionality every time i test this following issues i face , so strongly suggesting following improvements

BLOGS
1) After entering all the data when i click "SAVE" the message come on top of the editing windows is " Info! Successfully added the content" although the Publishing option is below (not everyone will understand that they should also need to PUBLISH...Keeping more steps is not good for UX)

Even if some user go below and click on PUBLISH button then the message comes 
"Info! Failed to publish
Thank you for posting the content. Pending Admin Approval. you will be notified once it is approved by admin." which make it more confusing

WE HAVE CLEAN & SIMPLE WAY TO SUBMIT "DISCUSSIONS" check that but because blog should have save option as well , following is the solution i thought should be more appropriate 

Solution:
- Keep All the option visible at First view itself SAVE | PUBLISH (both)......let user to decide what he want to do

- If User click on SAVE then the message should come as 
" Info! Successfully added the content, use PUBLISH option to submit it to admin"

- If User Click on PUBLISH then the message should come as
"Thank you for posting the content. Pending Admin Approval. you will be notified once it is approved by admin." 

THE MAIN THINGS IS in both BLOG & DISCUSSION when user click on PUBLISH 
then editing window should close automatically withing 10 seconds (so user can read the final message)..so that user will understand that his blog/discussion is get summited......Otherwise i just keep on wondering what to do next...fearing that may be his blog is not submitted and if he close the window his work will get lost in case

**Comment by arshadansari27** [comment]
I will do these changes once and for all and make it as simple as discussion. Also, the message that comes, I will put it below the button, so user can see it right there itself.

**Comment by YunusDange** [comment]
Yup...but remember the most important thing IN both BLOG & DISCUSSION is when user click on PUBLISH 
Editing window should close automatically withing 10 seconds (so user can read the final message before it get closed)..AND It should close automatically

**Comment by arshadansari27** [comment]
Why should it close. He may want to continue even after that. Make updations in case of spelling mistakes? I don't see why it should close?

**Comment by YunusDange** [comment]
He can always access it from "My Blog" for further editing...but logically
a user only save or submit when he is done with everything and want publish
it.

Being in initial days ..this functionality will make more sense in terms of
UX

**Comment by arshadansari27** [comment]
Alright then.. That is how we'll do it..

**Comment by YunusDange** [comment]
Although it Done and Wroking ....we two last issue remaining with this

1) now messages comes at the bottom for both (Blog and Discussions) but user should be guided towards that (means user should not have to scroll for that to see (this functionality is there when user click on "PUBLISH" option after saving but missing in following case
Blog: when user click on SAVE
Discussions: when user click on SUBMIT
Please check

2) You forget the issue mentioned above that is "Editing window should close automatically withing 10 seconds" after Clicking on PUBLISH (in Blog) & SUBMIT (in Discussions)

**Comment by arshadansari27** [comment]
1) On save I'm going to the bottom. So the user can see.

2) On Publish, I'm taking him to my blog section, where he can see the message as well as see his article in the list of his articles.

I think this solves the purpose. Do let me know what you think. If its ok. You can close this ticket.

**Comment by YunusDange** [comment]
NOT WORKING 1) On save user is not going to the bottom

WORKING PERFECTLY 2) On Publish, I'm taking him to my blog section..

NOT DONE (in DISCUSSION) .... Above things have done for BLOG.....but not done for DISCUSSIONS....Do the 2nd thing for discussion 
2) On Submit, take user to main Discussion section with the message
